### DISCUSSION TITLE
Deleted files from web ui, now can't reupload

### DISCUSSION THREAD

**Discussion by tfmm** [immich-app/immich#8272] [Q&A] [answered]
For some reason several recent uploads to immich were broken, unable to be viewed or show thumbnails.  I removed them from the web UI, but now when I attempt to reupload them from the web ui, they are detected as duplicates even though they were deleted from the trash. 

What can be done so i can get these upload?

**Comment by alextran1502** [comment]
It is in the trash page, you will need to remove it from the trash page to reupload it

**Reply by tfmm** [reply]
I did that, still being detected as a duplicate.

**Reply by alextran1502** [reply]
please make sure they are actually removed from trash, not recovered back to the timeline

**Reply by tfmm** [reply]
So I just checked the trash, and somehow they are back there. I attempted to delete them multiple times again, and upon a page refresh, they just come back to trash.

**Reply by alextran1502** [reply]
Did you use this button? 
![image](https://github.com/immich-app/immich/assets/27055614/8266e525-8814-49b3-9170-239f7d606262)

**Reply by tfmm** [reply]
I tried that initially, that did nothing, so I selected the affected images and used the delete button at the top of the screen.

**Reply by tfmm** [reply, marked as answer]
So it appears something was just angry in my stack of containers. I redeployed the whole compose stack, and now it appears to be working fine.
